exercises/concept/lasagna-master/.docs/instructions.md

```diff
@@ -49,8 +49,8 @@ It was amazing and had something special to it.
 The friend sent you the list of ingredients and told you the last item on the list is the "secret ingredient" that made the meal so special.
 Now you want to add that secret ingredient to your recipe as well.
 
-Write a function `addSecretIngredient` that accepts two `string` vectors of ingredients as parameters.
-The first parameter is the list your friend sent you, the second is the ingredient list of your own recipe.
+Write a function `addSecretIngredient` that accepts two `vector of strings` as parameters.
+The first one is a reference to the ingredient list of your own recipe, while the second one is the `vector of strings` of the ingredient list your friend sent you.
```

Comment:
That is a good clarification. We should let the users decide what to do. Both parameters can be references. For performance they both should be, and the second one should be _const_. 

I think Siebenschläfer and I had the cpp terms in _italics_ and parts from the actual code in `blocks`.

What do you think about that?

```suggestion
The first parameter is a _reference_ to the ingredients of your own recipe, and the second _vector_ of strings has the ingredients your friend sent you.
```

Reply:
With [this commit](4c09b8a7d3e11976157ada59ad8a58681a5ccad5), I added in the note the possibility of using a reference also for the friend's list.
The fact that it has not to be modified was already specified there.

I also accepted the suggestion for the style of the instructions.